src/main/java/cn/nukkit/block/BlockCandle.java

```diff
@@ -45,8 +45,7 @@ public BlockCandle(int meta) {
 
     @Override
     public boolean place(@Nonnull Item item, @Nonnull Block block, @Nonnull Block target, @Nonnull BlockFace face, double fx, double fy, double fz, @Nullable Player player) {
-
-        if (target.getId() == getId()) {
+        if ((target.getId() == getId()) || (getBlock(target.getLocation().add(0, 1, 0)).getId() == getId())) {
```

Comment:
Random `BlockCandle` fix, please revert.

Reply:
Oops, sorry. Thanks for the warning, I will revert now.